### PR TITLE
Add kube-burner's cluster-density & kubelet-density-heavy

### PR DESCRIPTION
**Pull request by rsevilla87**
cc: @chaitanyaenr @mohit-sheth @jtaleric 

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM!

File: jjb/dynamic/scale-ci_cluster-density.yml [hunk 1]
```diff
@@ -0,0 +1,168 @@
+- job:
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_cluster-density.yml:1]: We need to add a build script which reads the properties files and calls the cluster density job when selected to run as part of the pipeline. For example: https://github.com/openshift-scale/scale-ci-pipeline/pull/61/files.

Comment (rsevilla87) [reply jjb/dynamic/scale-ci_cluster-density.yml:1]: yup, WIP

File: README.md [hunk 2]
```diff
@@ -55,8 +55,8 @@ OpenShift install on GCP | Installer | Installs and configures OCP cluster | :he
 Tooling | Tooling | Sets up pbench-agents to collect perf data | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 Conformance | OCP/kubernetes e2e tests | Checks if the OCP cluster is sane | :heavy_check_mark: | :x: | :heavy_check_mark: |  
 Scaleup | scaleup | Scales up OCP cluster to a desired node count | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |  
-Node Vertical | Kubelet performance and Cluster Limits | Creates max pods per compute node | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |  
-Master Vertical | Control plane density | Creates bunch of objects to stress ApiServer, Etcd and Controller | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |  
+Kubelet density | Kubelet performance and Cluster Limits | Creates max pods per compute node | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```
Comment (chaitanyaenr) [inline README.md:58]: We might want to delete the nodevertical and mastervertical build scripts/properties files.

Comment (rsevilla87) [reply README.md:58]: I didn't want to break anything. We can remove then once we have all pipelines migrated.

Comment (chaitanyaenr) [reply README.md:58]: Sounds good.

**Review by mohit-sheth — APPROVED** [review]
LGTM, we can remove mastervert and nodevert bits from the Jenkinsfile.

**Comment by rsevilla87** [comment]
> LGTM, we can remove mastervert and nodevert bits from the Jenkinsfile.

ok, will open a new PR to remove them

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**